backend/fms_core/migrations/0040_v3_10_0.py

```diff
@@ -0,0 +1,58 @@
+from django.conf import settings
+from django.db import migrations, models
+from django.contrib.auth.models import User
+import reversion
+
+ADMIN_USERNAME = 'biobankadmin'
+
+
+def add_property_types_for_normalization(apps, schema_editor):
+    Protocol = apps.get_model("fms_core", "Protocol")
+    PropertyType = apps.get_model("fms_core", "PropertyType")
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+
+    with reversion.create_revision(manage_manually=True):
+        admin_user = User.objects.get(username=ADMIN_USERNAME)
+        admin_user_id = admin_user.id
+
+        reversion.set_comment("Add properties' type for Normalization")
+        reversion.set_user(admin_user)
+
+        PROPERTY_TYPES_BY_PROTOCOL = {
+            "Normalization": [
+                ("Volume", "str"),
```

Comment:
maybe call these "target volume" and "target concentration" or "final ..." or ... Just to make sure the content is clear enough for the user (and for us).